apollo-api/src/main/java/com/apollographql/apollo/api/Operation.java

```diff
@@ -44,6 +44,15 @@
    */
   @NotNull String operationId();
 
+  /**
+   * Parses provided GraphQL operation raw response
+   *
+   * @param response operation raw response to parse
+   * @param scalarTypeAdapters configured instance of custom GraphQL scalar type adapters
+   * @return parsed GraphQL operation {@link Response}
+   */
+  @NotNull Response<T> parse(@NotNull Map<String, Object> response, @NotNull ScalarTypeAdapters scalarTypeAdapters);
```

Comment:
How hard would it be to support a streaming API ? 
Something like:
`@NotNull Response<T> parse(@NotNull okio.Source source, @NotNull ScalarTypeAdapters scalarTypeAdapters);`

Reply:
It's not hard at all, but keep in mind Kotlin Native, until we get `Okio` as KN library we shouldn't use any JVM libs.

Reply:
Looks like `Okio` is multiplatform actually: https://square.github.io/okio/changelog/#version-230.
[Source](https://square.github.io/okio/2.x/okio/okio/-source/) is written as supporting native/js

Reply:
Will definitely take a look